### ISSUE TITLE
No focus indication on navigation buttons

### ISSUE THREAD

**Issue by becka11y** [gitpitch/gitpitch#287]
While I realize that the navigation arrows are keyboard accessible via space and arrow key press, they don't have a visible focus indicator when focused. This can be very confusing for keyboard users who use tab and shift-tab to navigate through the page.  I realize I can override the outline: 0 in my .yaml file but I would prefer not.  By default, the arrow buttons should have a focus indicator. Visible focus is required by the Web Content Accessibility Guidelines.

**Comment by becka11y** [comment]
Here are the Web Content Accessibility Guidelines (WCAG) 2.1: https://www.w3.org/TR/WCAG21/.  The specific success criterion is:  https://www.w3.org/TR/WCAG21/#focus-visible.  The keyboard only user needs to know where focus is located. The arrows don't need an outline all the time but do need more than a color change to indicate when they have focus.
